Utils/Functions/Net/Download/script.js

```diff
@@ -32,6 +32,10 @@ var Downloader = function () {
         that._localFile = localFile;
         return that;
     };
+    that.ifUpdateAvailable = function (ifUpdateAvailable) {
```

Comment:
Like I wrote in the description: It allows you to only download the file again if the file on the web server is newer than the one you have downloaded (i.e. it has been updated on the server).